libs/watt/package/modal/watt-modal.component.scss

```diff
@@ -18,6 +18,11 @@
   --watt-modal-min-height: auto;
   --watt-modal-height: calc(100vh - var(--watt-topbar-height, 0px));
   --watt-modal-content-padding: 0 var(--watt-space-ml);
+  border-radius: var(--watt-radius-l);
+}
+
+:host {
+  border-radius: 20px;
```

Comment:
suggestion: we just standardized the `border-radius`, seems a little soon to deviate from it. if "small, medium, large" is too limiting, then we could consider using numbers instead (--watt-radius-100, --watt-radius-200, --watt-radius-300, --watt-radius-400 etc). The numbers are not related to their actual size, but are only for "hierarchy".